src/global.css

```diff
@@ -88,4 +88,44 @@ a:hover {
 
 .cm-lineNumbers > .cm-gutterElement[err-line] {
 	background: #ecb2b2!important;
-}
\ No newline at end of file
+}
+
+/* Bad code by Lucas */
```

Comment:
remove comment if in jest, or at least specify why it is bad...

Reply:
ohhh I see - this was just to section off your changes (definitely think it makes more sense to put in separate file).